src/components/com_tjdashboard/administrator/views/widget/tmpl/edit.php

```diff
@@ -32,6 +32,16 @@
 	{
 		if (task == "widget.cancel" || document.formvalidator.isValid(document.getElementById("widget-form")))
 		{
+			if (task != "widget.cancel" && jQuery("#jform_params").val())
```

Comment:
Please create a custom validator for validating JSON input. Also, you are writing the value back to the field, the validator should not manipulate the value in anyway, just validate it. 

cc @thite-amol

Reply:
just validate it - done